src/routes/transactions/__tests__/controllers/list-incoming-transfers-by-safe.transactions.controller.spec.ts

```diff
@@ -129,6 +144,21 @@ describe('List incoming transfers by Safe - Transactions Controller (Unit)', ()
     const chainId = faker.string.numeric();
     const safeAddress = faker.finance.ethereumAddress();
     const chain = chainBuilder().with('chainId', chainId).build();
+    const safe = safeBuilder()
+      .with('address', '0x4127839cdf4F73d9fC9a2C2861d8d1799e9DF40C')
```

Comment:
Why do we need a specific ethereum address here?

Reply:
We don't 🙂 I'll change it to just rely on the builders.

I was just changing the JSONs for builder instances, and removing some redundant data, but I didn't take a deeper look into the logic of the tests, so maybe in some places there is redundant data that was present also before this PR was implemented. 

But yes, thanks for the suggestion, I'll also take the opportunity to remove redundant data where is possible 😄

Reply:
Changed in [0450df9](https://github.com/safe-global/safe-client-gateway-nest/pull/503/commits/0450df9f6fa71b7b2f5bd8b364e0fa0aedbdb2f3)